app/controllers/businessmatching/RegisterServicesController.scala

```diff
@@ -133,6 +113,39 @@ class RegisterServicesController @Inject()(val authConnector: AuthConnector,
         }
   }
 
+  private def formatBusinessActivities(data: BusinessMatchingActivities)
+                        (implicit ac: AuthContext, hc: HeaderCarrier) = {
+    (for {
+      isPreSubmission <- statusService.isPreSubmission
+      businessMatching <- businessMatchingService.getModel.value
+      savedModel <- updateModel(
+        businessMatching,
+        newModel(businessMatching.activities,
+          data,
+          isPreSubmission
+        ),
+        isMsb(data, businessMatching.activities)
+      )
+      _ <- maybeRemoveAccountantForAMLSRegulations(savedModel)
+      _ <- clearRemovedSections(
+        businessMatching.activities.getOrElse(
+          BusinessMatchingActivities(
+            Set()
+          )
+        ).businessActivities,
+        savedModel.businessActivities
+      )
+      isRemoving <- isRemovingActivity(
```

Comment:
isRemovingActivity is a clearer name, especially if we remove the helper function.